hazelcast/src/main/java/com/hazelcast/quorum/QuorumConfig.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.hazelcast.quorum.QuorumType.READ_WRITE;
+
+/**
+ * Contains the configuration for cluster quorum.
+ */
+public class QuorumConfig {
```

Comment:
Probably you need to provide a QuorumConfigReadonly like the rest fhe configs are using.